src/index.test.ts

```diff
@@ -48,7 +48,6 @@ describe("SkynetClient", () => {
     expect(client).toHaveProperty("registry");
     expect(client.registry).toHaveProperty("getEntry");
     expect(client.registry).toHaveProperty("getEntryUrl");
-    expect(client.registry).toHaveProperty("getEntryLink");
```

Comment:
I'm not sure I'm getting this right. Does this mean that we're dropping the method? I thought we were going to keep it and mark it as deprecated?